br/pkg/utils/backoff.go

```diff
@@ -22,7 +22,7 @@ const (
 	importSSTWaitInterval    = 40 * time.Millisecond
 	importSSTMaxWaitInterval = 10 * time.Second
 
-	downloadSSTRetryTimes      = 8
+	downloadSSTRetryTimes      = 24
```

Comment:
could you comment what's actual take time of the download for one download request?

Reply:
original retry waiting time: 1+2+4*6=27 seconds
and the caller of download use retry again, so the waiting time is 8*27 + 27 = 243 seconds.
and the download total use 8 minutes(400 seconds)